red-hat-data-services/odh-deployer
# Fix isv rearrange order

## Description

Follow up of https://issues.redhat.com/browse/RHODS-9226 introduced a bug where some of the shared tiles were misconfigured

## How Has This Been Tested?



1. Install the operator on a managed cloud
2. Check that starbust enterprise is not installed
3. Check that the rest of applications are there

<img width="2178" alt="Screenshot 2023-06-09 at 09 30 34" src="https://github.com/red-hat-data-services/odh-deployer/assets/16117276/46549507-b2f8-41d7-be41-28e0a480f839">

## Merge criteria:



- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-9226
- [X] Live build image: `quay.io/lferrnan/rhods-operator-live-catalog:1.28.0-deployer`
